gneiss/stats/composition.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+from skbio import DistanceMatrix
+
+
+def variation_matrix(X):
+    """ Calculate Aitchison variation matrix.
+
+    Parameters
+    ----------
+    X : np.array
```

Comment:
I think you need to provide a Pandas Dataframe to use methods like iloc. It's not working with an pure np.array for me.